### ISSUE TITLE
Monero daemon faild to start

### ISSUE THREAD

**Issue by AMbns01** [monero-project/monero-gui#4312]
My monero synced 138Gb data but now after start daemon there is an issue attch please help
![failed to start](https://github.com/monero-project/monero-gui/assets/168646452/3ead2a12-3c66-40c1-b13f-ca319576ed59)

**Comment by AMbns01** [comment]
yes dear i was waiting for your help yes i downloaded almost 138Gb files but the my memory ran out now i copied my data to my D drive there i have storage but my daemon not starting getting the above error

**Comment by selsta** [comment]
Did you set the blockchain location in Settings -> Node -> Blockchain directory?

**Comment by AMbns01** [comment]
yes i tried both the new directory also and by reseting it also

**Comment by AMbns01** [comment]
bro can you please help me in this

**Comment by selsta** [comment]
It is difficult to remotely figure out what the issue is. Can you just set a remote node?

You go to Settings -> Node, click on Remote Node and then on "Add remote node" and then enter

Address: node2.monerodevs.org
Port: 18089

then click on "Ok" and wait for it to sync.

You can find more remote nodes on monero.fail
